boards/shields/waveshare_ups/waveshare_pico_ups_b.overlay

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023 Joseph Yates
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	i2c1_default: i2c1_default {
+		group1 {
+			pinmux = <I2C1_SDA_P6>, <I2C1_SCL_P7>;
+			input-enable;
+                        input-schmitt-enable;
+			bias-pull-up;
+		};
+	};
+};
+
+pico_i2c1: &i2c1 {};
```

Comment:
Could you consider to create `boards/shields/waveshare_ups/boards/rpi_pico.overlay`.

Reply:
Yes, will get on that now :)